docs/conf/eu/2017/news/recap-retrospective.rst

```diff
@@ -0,0 +1,90 @@
+
+- On the first conference day (Monday), there was an unfortunate mixup where some vegetarian lunch options were missing. We became aware of this problem as lunch started and immediately discussed this with the venue, who acknowledged that this was a mistake on their part. We are continuously working with the venue to improve and broaden the menu options for as many dietary requirements as possible, and will continue to monitor the catering operations next year.
+
+- During the unconference sessions, there were times when the noise level in the room was uncomfortably high, thanks to the increased interest and excitement of our attendees in the unconference sessions. Although we couldn't make major changes to the space during the conference itself, we have made a careful note of this potential issue and will make the necessary adjustments to the venue layout so that we can accommodate all the sessions comfortably.
+
+We appreciate all your feedback, and strive to adapt and evolve our events to make everyone's experience a joyous one!
+
+Code of Conduct transparency report
+-----------------------------------
+
+Write the Docs is an inclusive, friendly, and welcoming community to all documentarians. As a part of our ongoing efforts to promote safe spaces in the tech fields, we apply a `Code of Conduct <http://www.writethedocs.org/code-of-conduct/>`_ to all of our conferences, meetups, and online spaces.
+
+Following the conference, here is an anonymized list of CoC-related incidents that occurred during the conference:
+
+- During the conference, an attendee tweeted a joke about inappropriate alcohol use with the #writethedocs hashtag. We asked the attendee to delete the tweet, which was done immediately.
+
+- One of the presenters made a joke during their talk that was considered discriminatory towards elderly people, parents, and/or women. We removed the mention from the presentation video before publishing it, and asked that the presenter refrain from these jokes in future Write the Docs events.
+
+- We received several reports of over-use of the word "guys" to refer to people in general. This was not confined to a single incident, and repeated among presenters and attendees. Barring specific people to address, we'd like to remind everyone to carefully consider which pronouns and other classifications to use when addressing a diverse group.
+
+As mentioned, this report is anonymized and published for the sake of maintaining transparency, and to encourage folks in our community to report incidents that might disrupt the atmosphere we are passionate about.
+
+Videos
+------
+
+All talk videos are now availble on our `playlist on our YouTube channel <https://www.youtube.com/playlist?list=PLZAeFn6dfHplBYPCwJt6ItkMDt7JSgUiL>`_.
```

Comment:
'now available in one playlist' (just so we're not repeating 'on our')

Reply:
Oops, good catch! I actually had different text in the link and when I flipped the sentence I forgot to fix it.